app/code/Magento/Sales/Model/Service/InvoiceService.php

```diff
@@ -90,7 +81,7 @@ public function __construct(
         \Magento\Sales\Model\Order\InvoiceNotifier $notifier,
         \Magento\Sales\Api\OrderRepositoryInterface $orderRepository,
         \Magento\Sales\Model\Convert\Order $orderConverter,
-        Json $serializer = null
+        JsonSerializer $serializer
```

Comment:
Hi, @ernesthernandez, due to [Magento backward-compatible guide](http://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/) we can't add required parameters to the constructor. Please revert your changes.